### PR TITLE
Add Milkomeda A1 configs

### PR DESCRIPTION
**Pull request by bartfish**
- [x] Mainnet

Not sure if the branch `base` is correct, hence please check this one too. cc @nicarq @rinor @NicolasDP

### CODE REVIEW

**Comment by NicolasDP** [comment]
Seems good to me

File: _data/chains/eip155-2002.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Milkomeda A1 Mainnet",
+  "chain": "milkALGO",
+  "icon": "milkomeda",
+  "network": "mainnet",
+  "rpc": [
+    "https://rpc-mainnet-algorand-rollup.a1.milkomeda.com",
+    "wss://rpc-mainnet-algorand-rollup.a1.milkomeda.com"
```
Comment (rinor) [inline _data/chains/eip155-2002.json]:
```suggestion
    "wss://rpc-mainnet-algorand-rollup.a1.milkomeda.com/ws"
```

**Review by rinor — APPROVED** [review]

**Review by nicarq — APPROVED** [review]

**Comment by SebastienGllmt** [comment]
Closing because there are ugly merge conflicts and this PR is to the wrong repo anyway

File: _data/chains/eip155-2002.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Milkomeda A1 Mainnet",
+  "chain": "milkALGO",
+  "icon": "milkomeda",
+  "network": "mainnet",
```
Comment (SebastienGllmt) [inline _data/chains/eip155-2002.json:5]: Just FYI that the `network` key was removed https://github.com/ethereum-lists/chains/pull/1508

**Pull request closed without merging**